src/llmcompressor/utils/helpers.py

```diff
@@ -1087,31 +1089,42 @@ def __exit__(self, _exc_type, _exc_val, _exc_tb):
 
 
 @contextlib.contextmanager
-def DisableQuantization(model: torch.nn.Module):
+def DisableQuantization(module: torch.nn.Module):
```

Comment:
```suggestion
def distable_quantization(...)
```
I do prefer module here

Reply:
@horheynm This comment is very confusing.

1. If you prefer `module`, then that's good because it has been changed to `module.
2. Why did you change the style of the function name? (it has a typo btw)

Reply:
1. module from model is better
2. You have a context manager that uses both capitalized and snakecase. Why is this capitalized?

Reply:
Usually capitalized context managers are for classes that have the entry and exit ex. ExitStack
https://asyncstdlib.readthedocs.io/en/stable/source/api/contextlib.html#asyncstdlib.contextlib.ExitStack

Reply:
> You have a context manager that uses both capitalized and snakecase. Why is this capitalized?

Please read the comment I just sent

Reply:
This doesnt explain anything